### PR TITLE
Bunk: Adapt project level build.gradle and gradle-wrapper.properties …

### PR DESCRIPTION
**Pull request by jashasweejena**
…to Android Studio 3.2

Make sure these boxes are checked before submitting your pull request.

- [x] My code follows the style guidelines of this project
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Mon Apr 30 10:50:12 IST 2018
+#Tue Oct 23 16:11:09 IST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4-all.zip
+distributionUrl=http://services.gradle.org/distributions/gradle-4.10.2-all.zip
```
Comment (abhijitparida) [inline gradle/wrapper/gradle-wrapper.properties]: Add `https` to the url.

**Comment by abhijitparida** [comment]
In `app/build.gradle`, remove `buildToolsVersion "27.0.3"`.

**Comment by jashasweejena** [comment]
I'll update with the required changes shortly. Android Studio actually started redownloading the gradle files abruptly. Once it finishes gradle sync I'll update this.

**Comment by abhijitparida** [comment]
I have [updated the build config](https://github.com/abhijitparida/bunk/commit/40628d88399d49d685c60d86a0e1c9f551653b33).

**Comment by jashasweejena** [comment]
Cool!

**Pull request closed without merging**